sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala

```diff
@@ -245,6 +245,10 @@ class SQLQueryTestSuite extends QueryTest with SharedSQLContext {
 
     (1 to 100).map(i => (i, i.toString)).toDF("key", "value").createOrReplaceTempView("testdata")
 
+    Seq((1, 1), (1, 2), (2, 1), (2, 2), (3, 1), (3, 2))
```

Comment:
The major differences are the data. They have different data distribution. For example, testData` does not have duplicate key values, but `testData2` has fewer rows and duplicate key values. `src1` has null but `src` does not have it. Your concern is valid. We should change the name; otherwise, it is hard to understand the reasons.